.teamcity/subprojects.json

```diff
@@ -1077,6 +1077,13 @@
     "functionalTests": false,
     "crossVersionTests": true
   },
+  {
+    "name": "unit-test-fixtures",
```

Comment:
This name is not giving me enough context to understand what this module is about. At first I thought it was something like `:internal-integ-testing` or something. Multi-level project names (like `:extensibility:unit-test-fixtures`) could clarify the intent here. If we don't want to do that, we should consider adopting prefixes, like `:extensibility-unit-test-fixtures`.

Reply:
It does live in the extensibility platform, so there is some context there. I guess the same argument could be made about many of the project names. I'm not sure we should solve a cross-cutting problem in an ad hoc way.

Reply:
Agreed that we don't have to solve this here. But as we add more and more subprojects, scoping will be more and more of a problem, so we should address it in a cross-cutting way.